src/Mod/Sketcher/Gui/Resources/Sketcher.qrc

```diff
@@ -223,28 +224,42 @@
         <file>icons/pointers/Sketcher_Pointer_Create_ArcSlot.svg</file>
         <file>icons/pointers/Sketcher_Pointer_Create_Box.svg</file>
         <file>icons/pointers/Sketcher_Pointer_Create_Box_Center.svg</file>
+        <file>icons/pointers/Sketcher_Pointer_Create_Box_3Points.svg</file>
+        <file>icons/pointers/Sketcher_Pointer_Create_Box_3Points_Center.svg</file>
         <file>icons/pointers/Sketcher_Pointer_Create_BSpline.svg</file>
+        <file>icons/pointers/Sketcher_Pointer_Create_BSplineByInterpolation.svg</file>
         <file>icons/pointers/Sketcher_Pointer_Create_Circle.svg</file>
         <file>icons/pointers/Sketcher_Pointer_Create_Ellipse.svg</file>
```

Comment:
No reason to keep this file. Alternatively you can just replace it by your new Sketcher_Pointer_Create_EllipseByCenter and not create a new `Sketcher_Pointer_Create_EllipseByCenter` file.

Reply:
I am not sure if this is used elsewhere

Reply:
In doubt you can do a ctrl-F to look through the whole project for the file name. On visual studio you can look in 'entire solution' which will look in all the source files.

Reply:
Btw I checked and `Sketcher_Pointer_Create_Ellipse `does not appear anywhere else in the source.

Reply:
For the terminal loving ones:
```grep -i -r Create_BSplineByInterpolation *``` returns nothing.

```
grep -i -r Create_Ellipse *
Mod/Sketcher/Gui/Resources/Sketcher.qrc:        <file>icons/pointers/Sketcher_Pointer_Create_Ellipse.svg</file>
Mod/Sketcher/Gui/DrawSketchHandlerEllipse.h:        return QString::fromLatin1("Sketcher_Pointer_Create_Ellipse");
```

What do you think is the good way forward here?

Reply:
ok remove then